Cargo.toml

```
@@ -11,6 +11,15 @@ arrayvec = { version = "0.7.1", default-features = false }
 [target.thumbv6m-none-eabi.dependencies]
 nanos_sdk = { git = "https://github.com/LedgerHQ/ledger-nanos-sdk.git" } 
 
+[target.nanos.dependencies]
+nanos_sdk = { git = "https://github.com/LedgerHQ/ledger-nanos-sdk.git" } 
+
+[target.nanosplus.dependencies]
+nanos_sdk = { git = "https://github.com/LedgerHQ/ledger-nanos-sdk.git" } 
+
+[target.nanox.dependencies]
+nanos_sdk = { git = "https://github.com/LedgerHQ/ledger-nanos-sdk.git" } 
+
```

Can we do this instead?
```suggestion
[target.'cfg(target_family = "bolos")'.dependencies]
nanos_sdk = { git = "https://github.com/LedgerHQ/ledger-nanos-sdk.git" }  

```

---

Done